assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss

```diff
@@ -232,6 +232,11 @@
 			outline-width: 1px;
 		}
 
+		&:active {
```

Comment:
@max-zu I prefer to make the smallest update. necessary, to avoid that we clash with existing styling e.g. from 3rd party plugins.

My question: do we need all of this styling to remove the blue color on slide?